### PR TITLE
Implement task updates from Github webhook

### PR DESCRIPTION
**Pull request by vikoivun**

### CODE REVIEW

File: helpt/urls.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from django.contrib import admin
 from rest_framework.routers import DefaultRouter
 
+from projects.adapters.github_hook import urls as github_urls
```
Comment (juyrjola) [inline helpt/urls.py]: I think `github.py` has not grown to be too big to include the hook code, as well. So move the contents of `github_hook.py` into `github.py`.

File: helpt/urls.py [hunk 2]
```diff
@@ -36,4 +37,5 @@
     url(r'^admin/', admin.site.urls),
     url(r'^v1/', include(router.urls, namespace='v1')),
     url(r'^$', front_page),
+    url(r'^hooks/github', include(github_urls, namespace='github_hook')),
```
Comment (juyrjola) [inline helpt/urls.py]: Should this have a trailing slash? Also, `namespace` is unnecessary here. If you want to name your view, name it in `github_urls`.

**Review by juyrjola — CHANGES_REQUESTED** [review]

File: helpt/urls.py (hunk 1 above)
Comment (vikoivun) [reply helpt/urls.py]: That causes a circular import of Workspace -> GitHubAdater -> Workspace, thus necessitating the usage of:
`
import django
Workspace = django.apps.apps.get_model('projects', 'Workspace')
`

I had that first, but it seemed _unclean_ somehow. Is that still the way it should be done?

File: helpt/urls.py (hunk 2 above)
Comment (vikoivun) [reply helpt/urls.py]: Yea, it probably should. Will remove namespace too.

File: helpt/urls.py (hunk 1 above)
Comment (juyrjola) [reply helpt/urls.py]: Hmm, maybe some kind of adapter registry would work better here. Let's go with your original solution for now.

**Pull request closed without merging**